src/Appwrite/Utopia/Response/Model/ProviderRepository.php

```diff
@@ -41,12 +41,6 @@ public function __construct()
                 'default' => false,
                 'example' => true,
             ])
-            ->addRule('runtime', [
-                'type' => self::TYPE_STRING,
```

Comment:
What do we do instead?

Reply:
ProviderRepository is (as of right now) only used as response model for endpoint that creates repository on GitHub.

Actual list of repositories either gives ProviderRepositoryRuntime, or ProviderRepositoryFramework - those extend this model and add the specific attributes